include/oneapi/dpl/dynamic_selection

```diff
@@ -0,0 +1,39 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_DYNAMIC_SELECTION
+#define _ONEDPL_DYNAMIC_SELECTION
+
```

Comment:
Every public header should have `#include "oneapi/dpl/internal/common_config.h"` at the very beginning, before #including any other file. This is necessary to provide a workaround for GCC 9/10 compilation errors with oneTBB, which should be enabled before including any standard library header.

Reply:
Done